Backend/Workez.API/Controllers/ShiftController.cs

```diff
@@ -0,0 +1,60 @@
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Workez.Application.DTOs;
+using Workez.Application.Services;
+using Workez.Infrastructure.Persistence;
+
+namespace Workez.API.Controllers;
+
+[ApiController]
+public class ShiftController : ControllerBase
+{
+    private readonly ShiftService _shiftService;
+
+    public ShiftController(ShiftService shiftService)
+    {
+        _shiftService = shiftService;
+    }
+
+    [HttpGet("api/shifts/")]
+    public async Task<IActionResult> GetAllShifts()
+    {
+        var shifts = await _shiftService.GetAllShiftsAsync();
+        return Ok(shifts);
+    }
+
+    [HttpGet("api/shifts/{id}")]
```

Comment:
```suggestion
    [HttpGet("api/shifts/{id}")]
[ProducesResponseType(StatusCodes.NotFound)]
[ProducesResponseType<Object>(StatusCodes.200Ok)
```
This ProducesResponseTypes are used to document the API in swagger. It gives information about what responses can the endpoint generate.